Meta/run.sh

```diff
@@ -212,7 +216,7 @@ fi
 
 if [ -z "$SERENITY_MACHINE" ]; then
     if [ "$SERENITY_ARCH" = "aarch64" ]; then
-        SERENITY_MACHINE="-M raspi3 -serial stdio"
+        SERENITY_MACHINE="-M raspi3b -serial stdio"
```

Comment:
According to https://www.qemu.org/docs/master/system/arm/raspi.html this is correct :+1: